Applications/Makefile

```diff
@@ -0,0 +1,43 @@
+include ../Makefile.common
+
+APPLICATIONS = About \
+			   Downloader \
+			   FileManager \
+			   FontEditor \
+			   IRCClient \
+			   Launcher \
+			   PaintBrush \
+			   ProcessManager \
+			   QuickShow \
+			   Taskbar \
+			   Terminal \
+			   TextEditor
+
+APPLICATIONS_BUILD = ${APPLICATIONS:=.out}
+APPLICATIONS_INSTALL = ${APPLICATIONS:=.install}
+APPLICATIONS_CLEAN = ${APPLICATIONS:=.clean}
+
+all: $(APPLICATIONS_BUILD)
+
+install: $(APPLICATIONS_INSTALL)
+
+clean: $(APPLICATIONS_CLEAN)
+
+%.out: %
```

Comment:
This approach is very shell like. There are no deps on the target. Also you can no longer type "make" to build one Application. Instead something like "make Launcher.out" must be ran in the dir above, which is pretty unusual.

Reply:
I agree with this comment, this makes it very confusing to build individual apps.

Reply:
I agree this doesn't work well with the current workflow of doing `make -c ../Applications/PaintBrush  && sudo ./sync && ./run` because it isn't consistent with other components.

But this is the kind of semantics I would like to have when I get the whole project build by a single makefile (which will know about all dependencies and build the project incrementally when fille changes).

Exemple:
```sh
# From the root of this repos

# These should never be used directly because has we will have a proper incremental build 
make PaintBrush         # Build PaintBrush
make PaintBrush.clean   # Clean PaintBrush
make PaintBrush.rebuild # Clean and build PaintBrush
make PaintBrush.install # Install PaintBrush to $(SYSROOT)

make all
sudo make sync
sudo make run-qemu # Normally this rule is the only one run when working on serenity,
                   # It will make all && sudo make sync && qemu -... 
```

All of that will work using `rules.mk` files scattered in the source tree containing individuals make recipes for each components the project.